std_msgs/Header header
geometry_msgs/Pose pose
geometry_msgs/Twist velocity
std_msgs/Float32 reward
std_msgs/Float32 thrust
std_msgs/Float32 prior
std_msgs/Bool end_of_ep
